spec/comment_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let(:user) { User.create(name: 'John') }
+  let(:post) { Post.create(author: user, title: 'Hello', text: 'This is my first post') }
+  subject { Comment.new(post:, author: user) }
+
+  describe '.update_comments_counter' do
+    it 'increments the comments_counter of the associated post' do
+      expect { Comment.update_comments_counter(post.id) }.to change { post.reload.comments_counter }.by(1)
+    end
+  end
+end
```

Comment:
- [ ] [OPTIONAL] Kindly move the `comment_spec.rb` test file into the model directory where other tests are located. This will help keep your test/spec directory more organized. You have done a good job with the test 👍🏽

Reply:
@Whoistolu Thank you Sir. I will kindly do that